Formula/rure.rb

```diff
@@ -0,0 +1,31 @@
+class Rure < Formula
+  desc "C API for RUst's REgex engine"
+  homepage "https://github.com/rust-lang/regex/tree/master/regex-capi"
+  url "https://github.com/rust-lang/regex/archive/1.5.4.tar.gz"
+  version "0.2.1"
+  sha256 "a91d5b3e1644a1b298ca4ac8e458d693ae268df7fd3307c6d5d12915b5bc3870"
+  license all_of: [
+    "Unicode-TOU",
+    any_of: ["Apache-2.0", "MIT"],
+  ]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build", "--manifest-path", "regex-capi/Cargo.toml", "--release"
+    include.install "regex-capi/include/rure.h"
+    lib.install Dir["target/release/librure.{a,dylib}"]
```

Comment:
```suggestion
    lib.install Pathname.glob("target/release/librure.*")
```
While `Dir` works for globbing, it's a little misleading since we're not really working with directories. We probably also want to install `librure.so` on Linux.

Reply:
Ah, that's a good point. When I did librure.* I got a warning that that would install `librure.d`, a dependency file that I don't think should be installed - is there a good way to say `librure.*` but not `librure.d`?

Reply:
```suggestion
    lib.install Pathname.glob("target/release/#{shared_library("librure")}")
    lib.install "target/release/librure.a"
```
It's probably just worth being more explicit for that.

Reply:
Figured it out:
```
lib.install Pathname.glob("target/release/librure.*") - [Pathname.new("target/release/librure.d")]
```
 but let me know if there's a different preferred way to write this.